ipset_linux_test.go

```diff
@@ -76,14 +77,22 @@ func TestParseIpsetListResult(t *testing.T) {
 	}
 	expectedMAC := net.HardwareAddr{0xde, 0xad, 0x0, 0x0, 0xbe, 0xef}
 	if !bytes.Equal(ent.MAC, expectedMAC) {
-		t.Errorf("expected MAC for first entry to be %s, got %s", expectedMAC.String(), ent.MAC.String())
+		t.Errorf(
+			"expected MAC for first entry to be %s, got %s",
+			expectedMAC.String(),
+			ent.MAC.String(),
+		)
```

Comment:
I'm not a maintainer on this repo, but I'd prefer the changes to be reverted; keeping these lines as a single line makes it easier to "skimp" over them when reading the code ("we're returning an error"); breaking it up over multiple lines doesn't _really_ improve readability, and introduces code-churn, which we should avoid.